components/Sales/Auction/CardFooter.tsx

```diff
@@ -36,7 +36,7 @@ const SaleAuctionCardFooter: VFC<Props> = ({
       px={4}
       fontSize="sm"
       fontWeight="semibold"
-      href={`/tokens/${assetId}${!isOwner && '/bid'}`}
+      href={`/tokens/${assetId}${!isOwner ? '/bid' : ''}`}
```

Comment:
In these cases it's always good to simplify the condition
```suggestion
      href={`/tokens/${assetId}${isOwner ? '' : '/bid'}`}
```

Similar for others

Reply:
not related to this PR anymore